src/StringTools.UnitTests/SpanBasedStringBuilder.Simple_Tests.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET35_UNITTEST
+extern alias StringToolsNet35;
+#endif
+
+using System.Collections.Generic;
+
+using Shouldly;
+using Xunit;
+
+#if NET35_UNITTEST
+using StringToolsNet35::Microsoft.StringTools;
+#endif
+
+namespace Microsoft.StringTools.Tests
```

Comment:
I don't know if this is possible, but SpanBasedStringBuilder and SpanBasedStringBuilder.Simple are supposed to work the same, right? Is there a way to parameterize a test class and invoke the exact same tests with each?

Reply:
Yes! With just a few more #ifdef's there can be only one SpanBasedStringBuilder_Tests.cs running against both 3.5 and full implementations. I am really glad that you took the time to review this PR, thank you. Fixed.